manifests/alias.pp

```diff
@@ -7,45 +7,20 @@
 #     }
 #
 define kmod::alias(
-  $source,
-  $ensure = 'present',
-  $file   = '/etc/modprobe.d/aliases.conf',
+  $modulename,
+  $ensure     = 'present',
+  $file       = "/etc/modprobe.d/${title}.conf",
+  $aliasname  = $title,
```

Comment:
Short answer, I like it more - but everyone is torn on the subject, so I refer to the PL docs....

"$name defaults to the value of $title, but users can optionally specify a different value when they declare an instance. This is only useful for mimicking the behavior of a resource with a namevar, which is usually unnecessary. If you are wondering whether to use $name or $title, use $title."